CHANGELOG.md

```diff
@@ -33,6 +33,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - #868 - ignore the `CARGO` environment variable.
 - #867 - fixed parsing of `build.env.passthrough` config values.
 
+<!-- last-header -->
+
```

Comment:
the changelog needs to have a mention of `.changesets` since its no longer a live document, `.changes` is

Reply:
This has now been documented as follows:

```markdown
# Change Log

All notable changes to this project will be documented in this file. This is an automatically-generated document: entries are added via changesets present in the `.changes` directory.
This project adheres to [Semantic Versioning](http://semver.org/).
```